Lib/codeop.py

```diff
@@ -87,7 +87,7 @@ def _maybe_compile(compiler, source, filename, symbol):
     # Catch syntax warnings after the first compile
     # to emit SyntaxWarning at most once.
```

Comment:
This comment is now outdated. It's not only about SyntaxWarning, but any warning, including DeprecationgWarning.

```suggestion
    # to emit warnings (SyntaxWarning, DeprecationWarning) at most once.
```